subgraphs/apeswap/README.md

```diff
@@ -0,0 +1,73 @@
+# ApeSwap Subgraph
+## Calculation Methodology v0.0.1
```

Comment:
We should change the version to v1.0.0 if it's completed, in order to differentiate from a version that's still a work in progress